api/v1alpha1/spinapp_types.go

```diff
@@ -80,6 +80,9 @@ type SpinAppSpec struct {
 
 	// Resources defines the resource requirements for this app.
 	Resources Resources `json:"resources,omitempty"`
+
```

Comment:
@calebschoepp i thought we were going to put these on the Executor because they're mostly common to a set of apps, not an individual app?

Reply:
Yep we were, this just got mixed up in the communcation between Andrew and I.

@asteurer let's move this over to `spinapp_executor_types.go`. This will change how you'll have to pull the value in the operator.